app/screens/settings-screen/lnurl-screen.tsx

```diff
@@ -10,6 +10,7 @@ import { palette } from "../../theme/palette"
 
 import type { ScreenType } from "../../types/jsx"
 import type { RootStackParamList } from "../../navigation/stack-param-lists"
+import { LN_PAGE_DOMAIN } from "../../constants/support"
```

Comment:
```suggestion
import { GALOY_PAY_DOMAIN } from "../../constants/support"
```

We renamed lnpage to galoy-pay sometime ago

Reply:
> Are you saying you would like me to rename the constant across the whole app in this PR?

That is correct. It's only used in two more places -> https://github.com/GaloyMoney/galoy-mobile/search?q=ln_page_domain so I think it makes sense to include that change in this PR since it touches that code anyway

Reply:
I've pushed the requested change.